cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -64,7 +63,9 @@
 import org.mortbay.jetty.Request;
 import org.mortbay.jetty.handler.AbstractHandler;
 import org.json.simple.JSONObject;
-
+import com.telefonica.iot.cygnus.handlers.CygnusHandler;
```

Comment:
That was already imported before, only moved from its original place (line 32) without no reason (Perhaps a mistake and a fix later)